include/fi_file.h

```diff
@@ -46,7 +47,6 @@ extern "C" {
 
 int fi_read_file(const char *dir, const char *file, char *buf, size_t size);
 int fi_poll_fd(int fd, int timeout);
-int fi_fd_nonblock(int fd);
```

Comment:
Since the function prototype is common between platforms, wouldn't it make more sense to leave this prototype here and delete the duplicate one from `include/windows/osd.h` instead of moving it down to `include/unix/osd.h`?